test/client/script.js

```diff
@@ -409,4 +409,20 @@ describe(`script cases`, () => {
             throw new Error(`Expected locale to be ${ expectedLocale }, got ${ receivedLocale }`);
         }
     });
+
+    it('should return computed locale when lang is zh_Hant', () => {
+        const expectedLocale = 'zh_Hant';
+
+        const url = insertMockSDKScript({
+            query: {
+                'locale': 'zh_HK'
+            }
+        });
+
+        const localeObject = getLocale();
```

Comment:
```suggestion
        const { lang: expectedLang } = getLocale();
```

Reply:
Please use `expectedLang`/`receivedLang` for naming.

Reply:
improved, thanks!